modules/nodes-pods-autoscaling-custom-rn-2172.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```

Comment:
PEER REVIEW: No change to text. Copied from the release note assembly to a new module and bumped up the heading levels. No change to the text.